be/src/vec/aggregate_functions/aggregate_function.h

```diff
@@ -228,6 +231,18 @@ class IAggregateFunctionHelper : public IAggregateFunction {
         }
     }
 
+    void streaming_agg_serialize(const IColumn** columns, BufferWritable& buf,
```

Comment:
Whether the performance improvement mainly comes from memory reuse?
Could you add some test results about how much performance improvement compared with the original one.
May be the improvement.